notagain/lua/notagain/pac_server/autorun/dev_helpers.lua

```diff
@@ -0,0 +1,8 @@
+function Say(string)
+	string.Replace( string, '"', '\"' )
+	if CLIENT then
```

Comment:
I agree with the others, one check on run, not every time the function is called.